src/pages/Privacy-Policy/Privacy.jsx

```diff
@@ -1,11 +1,322 @@
+            </p>
+            <p>
+              We and our affiliates will share/sell some or all of your personal
+              data with another business entity should we (or our assets) plan
+              to merge with, or be acquired by that business entity, or undergo
+              reorganization, amalgamation, or restructuring of business. Should
+              such a transaction occur, that another business entity (or the new
+              combined entity) will be required to follow this Privacy Policy
+              with respect to your personal data.
+            </p>
+          </section>
+          <section>
+            <h2>Links to Other Sites</h2>
+            <p>
+              Our Platform may provide links to other websites or applications
+              that may collect personal data about you, and you will be governed
+              by their privacy policies. VigyBag shall not be responsible for
+              the privacy practices or the content of their privacy policies,
+              and we request you to read their privacy policies prior to
+              disclosing any information.{" "}
+            </p>
+          </section>
+          <section>
+            <h2>Security Precautions</h2>
+            <p>
+              We maintain reasonable physical, electronic, and procedural
+              safeguards to protect your personal data from unauthorized access
+              or disclosure. Whenever you access your account information, we
+              offer the use of a secure server. Once your data is in our
+              possession, we adhere to reasonable security guidelines,
+              protecting it against unauthorized access.
+            </p>
+          </section>
+          <section>
+            <h2>Your Consent</h2>
+            <p>
+              By using the Platform and/or by providing your information, you
+              consent to the collection and use of the information you disclose
+              on the Platform in accordance with this Privacy Policy, including
+              but not limited to your consent for sharing your information as
+              per this privacy policy.
+            </p>
+            <p>
+              If we decide to change our Privacy Policy, we will post those
+              changes on this page so that you are always aware of what
+              information we collect, how we use it, and under what
+              circumstances we disclose it.
+            </p>
+          </section>
+          <section>
+            <h2>Grievance Officer</h2>
+            <p>
+              In accordance with the Information Technology Act, 2000 and the
+              rules made thereunder, the name and contact details of the
+              Grievance Officer are provided below:
+              <p>
+                <b>Name:</b> Vivek Prajapati
+              </p>
+              <p>
+                <b>Email:</b> <a className="a">grievanceofficer@vigybag.com</a>{" "}
```

Comment:
check this line for {" "}

Reply:
why are you not make this for mail:
see below code 

<a className="a" href="mailto:grievanceofficer@vigybag.com">grievanceofficer@vigybag.com</a>